docker-compose.yml

```diff
@@ -55,6 +56,7 @@ services:
       - DEBUG=1
       - IN_DOCKER=1
       - DISABLE_SENTRY=1
+      - DOCKER_REDIS=1
```

Comment:
does this need to be enabled in the worker as well?

Reply:
I don't think so - all the websocket and async functionality works (for me) as-is

Reply:
Just tried it out, adj and room allocations don't seem to work without it, though side editing and checkins work fine.